plugin/src/main/java/org/wildfly/plugin/provision/ApplicationImageInfo.java

```diff
@@ -77,9 +77,10 @@ public class ApplicationImageInfo {
     protected String password;
 
     /**
-     * The binary used to build and push images (default is "docker").
+     * The binary used to build and push images. The default will attempt to be determined. First we'll check if
+     * {@code podman} is available, if not {@code docker} will be used.
      */
-    protected String dockerBinary = "docker";
+    private String dockerBinary;
```

Comment:
Let's find better name even for internals, something like `containerTool`, `containerBinary`, `podBinary`, etc. Ideas?

Reply:
+1, that is what I'm doing. I'm just using `binary` in the `ApplicationImageInfo` as it is typically referred to as "image" so "binary" just seemed to make sense.

Reply:
Gotcha, oh well...